website/docs/user-guides/my-account.md

```diff
@@ -0,0 +1,42 @@
+---
+id: my-account
+title: My Account
+sidebar_label: My Account
+---
+
+## Introduction
+
+Under the `My Account` tab, you can change your personal details such as the email, full name, and password.
+
+> Note: The username can’t be changed as it is unique.
+
```

Comment:
A `Before you begin` section for this one as well

Reply:
`My account` has no relevant concept... :thinking: